complex/ft_test.c

```diff
@@ -251,6 +270,30 @@ void ft_next_iov_cnt(struct ft_xcontrol *ctrl, size_t max_iov_cnt)
 		ctrl->iov_iter = 0;
 }
 
+int ft_is_valid_atomic_op(size_t *count)
```

Comment:
The name of the function (is_valid) implies a boolean return (0 = false, 1 = true), but the behavior is the opposite (is_valid = 0 = true).  Maybe switch to using the fi_query_atomic() call, and check that the size is the same as well.  Then rename to ft_query...  Note that a size mismatch doesn't necessarily indicate an error, except for the fixed size types.